x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/add_external_connector/external_connector_logic.test.ts

```diff
@@ -108,6 +108,18 @@ describe('ExternalConnectorLogic', () => {
       });
     });
 
+    describe('saveExternalConnectorConfigError', () => {
+      it('turns off the button loading flag', () => {
+        mount({
+          buttonLoading: true,
+        });
+
+        ExternalConnectorLogic.actions.saveExternalConnectorConfigError();
+
+        expect(ExternalConnectorLogic.values.buttonLoading).toEqual(false);
```

Comment:
Nitpick: As an effort to make assertions consistent, we should check whole object instead of a value. I see the other tests on this file are doing same as you did it though. https://github.com/elastic/enterprise-search-team/issues/1230

```javascript
expect(ExternalConnectorLogic.values).toEqual({
  ...DEFAULT_VALUES, 
  buttonLoading: false,
});
```

Reply:
Ah, good nitpick! Changed all the tests in this file to match this approach.